### PR TITLE
turning off compile on loss function

### PR DESCRIPTION
**Pull request by tianyu-l**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #755

TBD: I'll create a pytorch issue with ways to repro errors

### CODE REVIEW

**Review by awgu — APPROVED** [review]
tragic but must be done
